hi-ml-cpath/src/health_cpath/utils/girder.py

```diff
@@ -537,17 +618,26 @@ def upload(
         help="Just log into the DSA and exit. Useful to ensure connection to the DSA from current host",
     )
     parser.add_argument(
-        "--rescale",
-        action="store_true",
-        help="Rescale attention values between 0 and 1 to maximize heatmaps contrast",
+        "--no-rescale",
+        action="store_false",
+        default=True,
+        help="Do not rescale attention values. By default, attention values are scaled to such that the range "
+             "between min and max fills the colormap.",
     )
     parser.add_argument(
         "--colormap",
         type=str,
         choices=plt.colormaps(),
-        default="Greens",
+        default="Spectral_r",
```

Comment:
Why Spectral? I think we should use a perceptually uniform colormap. More info:
- [How The Rainbow Color Map Misleads](https://eagereyes.org/basics/rainbow-color-map)
- [A dangerous rainbow: Why colormaps matter](https://blogs.mathworks.com/headlines/2018/10/10/a-dangerous-rainbow-why-colormaps-matter/)
- [Why We Use Bad Color Maps and What You Can Do About It](https://www.kennethmoreland.com/color-advice/BadColorMaps.pdf)
- [Choosing Colormaps in Matplotlib](https://matplotlib.org/stable/tutorials/colors/colormaps.html)

Reply:
We wanted the colormap to match what we use in the training code.

Reply:
Makes sense. My—predictable—suggestion is to change the training code :)

Reply:
[As opposed to this default]